.github/workflows/tests.yml

```diff
@@ -21,3 +21,31 @@ jobs:
         run: bundle exec fastlane tests
       - name: Codecov
         uses: codecov/codecov-action@v1.0.13
+      - name: "Upload SnapshotFailedTests"
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: PolaUITests-SnapshotFailedTests
+          path: BuyPolish/PolaUITests/SnapshotFailedTests/*.png
+      - name: "Upload PolaUITests-SnapshotFailedTests"
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: PolaUITests-SnapshotFailedTests
+          path: BuyPolish/PolaUITests/SnapshotFailedTests/*
```

Comment:
Dwa razy leci SnapshotFailedTests, wyżej jest `SnapshotFailedTests/*.png`
Czy jeśli na górze jest 
```
    defaults:
      run:
        working-directory: BuyPolish
```
To czy `BuyPolish` w ścieżce nie powinno być pominięte?
Nie napisałem tego w issue, ale snapshoty są wrzucane w ten sposób:
`SnapshotFailedTests/nazwaKlasyTestowej/nazwa_testu_fail.png`
Czy `SnapshotFailedTests/*` złapie zagnieżdzone w nim katalogi czy będzie potrzebny `SnapshotFailedTests/**/`

Reply:
> Dwa razy leci SnapshotFailedTests, wyżej jest SnapshotFailedTests/*.png
Popprawiłem

> To czy BuyPolish w ścieżce nie powinno być pominięte?

Wydaje mi się, że nie.  To by było bardzo dziwne, ale sprawdzę w wolnej chwili.

> Czy SnapshotFailedTests/* złapie zagnieżdzone w nim katalogi czy będzie potrzebny SnapshotFailedTests/**/

Skasowałem całkowicie wildcarda. Bez niego będzie cały katalog wrzucany, co nie powinno stanowić w naszym wypadku problemu.